### DISCUSSION TITLE
Review of ZhiPu Copilot

### DISCUSSION THREAD

**Discussion by AlexeyMatveev686** [ONLYOFFICE/onlyoffice.github.io#273] [Polls]
<img src="https://raw.githubusercontent.com/ONLYOFFICE/onlyoffice.github.io/master/sdkjs-plugins/content/zhipu/resources/store/icons/icon%402x.png">

**Comment by wer5lcy** [comment]
Good first version!
The copilot addon is especially good considering its accessability and support of Chinese language & localized knowledge base.  Please enable the add-on not only in document editor, but also presents and spreadsheet editors too.

**Reply by devount** [reply]
hello,the zhipu apikey format has changed，so the ZhiPu copilot plugin cannot work now,can you fix it? 
thanks very much

**Reply by AlexeyMatveev686** [reply]
@brick-pid Please, look it.

**Reply by brick-pid** [reply]
Ok, I'll take a closer look!

**Comment by devount** [comment]
您好，我已收到您的邮件，我会尽快处理。

**Reply by Misokados** [reply]
你好！请问具体是指什么内容？

**Comment by VladimirProkofiev52** [comment]
Hello!
I’ve noticed an issue and wanted to report it. When I launch the ZhiPu Copilot plugin, it adds a “ZhiPu Copilot” item to the right-click context menu. However, when I click on that item, it opens a submenu that is completely empty—there’s literally nothing in it. I hope this issue can be fixed. Thank you very much!

**Comment by devount** [comment]
您好，我已收到您的邮件，我会尽快处理。

**Reply by Misokados** [reply]
你好！请问具体是指什么内容？

**Comment by devount** [comment]
您好，我已收到您的邮件，我会尽快处理。
